web/modules/custom/joinup_core/joinup_core.install

```diff
@@ -62,6 +62,28 @@ function joinup_core_requirements($phase) {
         ];
       }
     }
+
+    // If there are forward published revisions in any entity, this will come up
+    // with this query. This is to ensure that the issue with the forward
+    // published revisions not being the latest published revision, is caught
+    // early if it happens again.
```

Comment:
This comment is assuming that everybody who will ever see this code will be familiar with "the issue with the forward published revisions not being the latest published revision".

It is allowed in core to publish older revisions, so can you explain in this comment exactly why Joinup cannot deal with this situation?